Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -6,11 +6,18 @@ import NIOHTTP1
 import NIOSSL
 import NIOWebSocket
 @testable import WebSocketKit
+import CompressNIO
 
 final class WebSocketKitTests: XCTestCase {
+    var elg: EventLoopGroup!
```

Comment:
Do we need this here? It's not created in `setUp` below. We seeem to have 2 `setUp` methods now. Can we make it one?

Reply:
sigh. yah that might have  happened when I merged we probably dont want two setup methods.